### PR TITLE
Replace most jQuery methods

### PR DESCRIPTION
**Pull request by oskarrough**
A work in progress to replace jQuery. See https://github.com/intercom/ember-href-to/issues/2

I had to polyfill classList but perhaps this is where jQuery should be kept? Same with the click events.

### CODE REVIEW

File: app/instance-initializers/ember-href-to.js [hunk 1]
```diff
@@ -11,17 +21,17 @@ function _getNormalisedRootUrl(router) {
 export default {
   name: 'ember-href-to',
   initialize: function(applicationInstance) {
-    let router = applicationInstance.container.lookup('router:main');
-    let rootURL = _getNormalisedRootUrl(router);
-    let $body = Em.$(document.body);
+    const router = applicationInstance.container.lookup('router:main');
+    const rootURL = _getNormalisedRootUrl(router);
+    const $body = Em.$(document.body);
```
Comment (GavinJoyce) [inline app/instance-initializers/ember-href-to.js]: I think these should be `let`s: http://madhatted.com/2016/1/25/let-it-be

**Comment by GavinJoyce** [comment]
@oskarrough thanks for working on this. I'm not sure if there is much value in removing jQuery however, especially as this addon is likely to be added to the ember framework at some point as [`url-for`](https://github.com/emberjs/rfcs/pull/92). I think the original issue should be closed.

**Comment by oskarrough** [comment]
Sure, I agree — feel free to close the pull request if you'd rather skip it. Otherwise I'll just change the const/let. Let me know.

**Comment by GavinJoyce** [comment]
Thanks, I'll close the issue and PR. Thanks for contributing :beers:

**Pull request closed without merging**